### ISSUE TITLE
Question about the computational performance of KDL on redundant kinematic chains

### ISSUE THREAD

**Issue by zichunxx** [orocos/orocos_kinematics_dynamics#423]
Hi!

I‘ve been learning to use `KDL` recently and have some confusion.

`KDL` is developed based on the jacobian pseudo-inverse and theoretically decreases computational efficiency as the degree of freedom (DOF) of the kinematic chain increases. 

However, as the comparison results showed in [trac-ik](https://bitbucket.org/traclabs/trac_ik/src/master/), the performance of `KDL` on a 15-DOF robot arm is better than that on a 7-DOF Kuka arm.

As I understand it, isn't the inverse of the Jacobian matrix more difficult to solve as the DOF increases?

Thanks! Answers would be appreciated!

**Comment by jrutgeer** [comment]
It says it in the text:
"Note on timings: The timings provided include both successful and unsuccessful runs. When an IK solution is not found, the numerical IK solver implementations will run for the full timeout requested, searching for an answer; thus for robot chains where KDL fails much of the time (e.g., Jaco-2), the KDL times are skewed towards the user requested timeout value (here 5 ms)."

In other words: a test with a low success rate has many unsuccessful runs (at 5 ms) and not many successful ones (e.g. at 0,6ms) so the average will be high.
